<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleObjectItemWidget</class>
 <widget class="QWidget" name="SingleObjectItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="single_object_setting_tab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(114, 114, 114);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Geometry">
    <attribute name="title">
     <string>Geometry</string>
    </attribute>
    <widget class="QCheckBox" name="visibility_checkbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Visibility</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="geometry_type_box">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Geometry type</string>
     </property>
     <widget class="QComboBox" name="geometry_type_combobox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="open_object_file_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Open object file</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="polygon_options_box">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>151</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>Polygon options</string>
     </property>
     <widget class="QCheckBox" name="smoothing_checkbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable smoothing</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="nagata_patch_checkbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Use Nagata patch</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Transformation">
    <attribute name="title">
     <string>Transformation</string>
    </attribute>
    <widget class="QFrame" name="transformation_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>311</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Material">
    <attribute name="title">
     <string>Material</string>
    </attribute>
    <widget class="QGroupBox" name="surface_scattering_box">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Surface scattering</string>
     </property>
     <widget class="QComboBox" name="surface_scattering_combobox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="emitter_box">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Emitter</string>
     </property>
     <widget class="QComboBox" name="emitter_combobox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="light_source_checkbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
